pages.zh/common/etcd.md

```diff
@@ -0,0 +1,20 @@
+# etcd
+
+> 分布式，可靠的键值存储，用于分布式系统中存储最关键的数据。
+> 更多信息见：https://etcd.io>.
```

Comment:
Not sure why the bot isn't flagging this 🤔
```suggestion
> 更多信息见：<https://etcd.io>.
```

Reply:
Maybe the regular expression has a bug?